spec/requests/v1/recipes_spec.rb

```diff
@@ -3,11 +3,47 @@
 require 'swagger_helper'
 
 RSpec.describe 'V1::Recipes', type: :request do
+  let(:user_credentials) { create(:user).create_new_auth_token }
+  let(:HTTP_ACCESS_TOKEN) { user_credentials['access-token'] }
+  let(:HTTP_TOKEN_TYPE) { user_credentials['token-type'] }
+  let(:HTTP_CLIENT) { user_credentials['client'] }
+  let(:HTTP_EXPIRY) { user_credentials['expiry'] }
+  let(:HTTP_UID) { user_credentials['uid'] }
+
   path '/v1/recipes' do
     post 'Creates a recipe' do
       tags 'Recipes'
       description 'Creates an instance of Recipe.'
       consumes 'application/json'
+      parameter name: :HTTP_ACCESS_TOKEN,
```

Comment:
is there a way of extracting these parameters, clutters up the test

Reply:
Not at this point